branches_num:  4
batch_sum:  4456
batch_sum:  17339
batch_sum:  21096
batch_sum:  24388
batch_sum:  21096
batch_sum:  24388
batch_sum:  21096
batch_sum:  24388
batch_sum:  21096
batch_sum:  24388
Time: 187.235053897 (s)
blocks_num:  16138
branches_num:  8
batch_sum:  8912
batch_sum:  33654
batch_sum:  42237
batch_sum:  47938
batch_sum:  42237
batch_sum:  47938
batch_sum:  42237
batch_sum:  47938
batch_sum:  42237
batch_sum:  47938
Time: 201.823445035 (s)
blocks_num:  20783
branches_num:  12
batch_sum:  13368
batch_sum:  50993
batch_sum:  63333
batch_sum:  72326
batch_sum:  63333
batch_sum:  72326
batch_sum:  63333
batch_sum:  72326
batch_sum:  63333
batch_sum:  72326
Error happens when executing Parrot program:  <class 'KeyboardInterrupt'> KeyboardInterrupt()
Traceback:  Traceback (most recent call last):
  File "/home/v-chaofanlin/ParrotServe/parrot/program/vm.py", line 313, in running_scope
    yield
  File "/home/v-chaofanlin/ParrotServe/parrot/program/vm.py", line 359, in run
    loop.run_until_complete(coroutine)
  File "/home/v-chaofanlin/miniconda3/envs/parrot/lib/python3.10/asyncio/base_events.py", line 636, in run_until_complete
    self.run_forever()
  File "/home/v-chaofanlin/miniconda3/envs/parrot/lib/python3.10/asyncio/base_events.py", line 603, in run_forever
    self._run_once()
  File "/home/v-chaofanlin/miniconda3/envs/parrot/lib/python3.10/asyncio/base_events.py", line 1871, in _run_once
    event_list = self._selector.select(timeout)
  File "/home/v-chaofanlin/miniconda3/envs/parrot/lib/python3.10/selectors.py", line 469, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
KeyboardInterrupt

Time: -6.0 (s)
blocks_num:  24097
branches_num:  16
batch_sum:  17824
batch_sum:  67308
batch_sum:  84474
batch_sum:  95876
batch_sum:  84474
batch_sum:  95876
batch_sum:  84474
batch_sum:  95876
batch_sum:  84474
batch_sum:  95876
